src/js/catblocks_msgs.js

```diff
@@ -26,8 +26,13 @@ Blockly.CatblocksMsgs.setLocale = function(locale, filesLocation = undefined) {
       return Promise.resolve();
     }
   } else {
-    console.warn('Fallback to default language and ignoring unrecognized locale: ' + locale);
-    return Blockly.CatblocksMsgs.setLocale(Blockly.CatblocksMsgs.currentLocale_);
+    const localeWithoutSuffix = locale.substring(0, locale.indexOf('_'));
```

Comment:
if language is unknown, we remove the suffix. Then we try again and if it is still unknown then the locale is set to the default language 'en'.